orchagent/portsorch.cpp

```diff
@@ -2755,6 +2772,12 @@ sai_status_t PortsOrch::removePort(sai_object_id_t port_id)
 
     removePortSerdesAttribute(port_id);
 
+    for (auto queue_id : port.m_queue_ids)
```

Comment:
Maybe UT hangs here.

Reply:
Hi
Can you share more details? I didn't see such errors locally nor in sonic-swss azure pipeline
Thanks

Reply:
The advancing submodule PR for swss https://github.com/sonic-net/sonic-buildimage/pull/15188 passed mock test.
For other PRs, the new swss (with my commits) won't be included until it is merged, right?

Reply:
offline discussed with @liushilongbuaa. the failure is not relevant to the PR since the PR is not in that image.
He will check it offline.